Makefile

```diff
@@ -1,6 +1,8 @@
 PKG := github.com/lightningnetwork/lnd
 ESCPKG := github.com\/lightningnetwork\/lnd
 MOBILE_PKG := $(PKG)/mobile
+TOOLS_DIR := tools
```

Comment:
Is there a specific reason for putting all the tools in that sub folder instead of the user's `$GOPATH/bin` folder? That makes them a bit harder to use as they're likely not in the `$PATH` variable anymore. And I assume the unit and/or integration tests will fail because of this.